hypothesis-python/src/hypothesis/extra/ghostwriter.py

```diff
@@ -906,22 +908,35 @@ def magic(
                 ]
                 if pkg and any(getattr(f, "__module__", pkg) == pkg for f in funcs):
                     funcs = [f for f in funcs if getattr(f, "__module__", pkg) == pkg]
-            for f in funcs:
-                try:
-                    if (
-                        (not is_mock(f))
-                        and callable(f)
-                        and _get_params(f)
-                        and not isinstance(f, enum.EnumMeta)
-                    ):
-                        functions.add(f)
-                        if getattr(thing, "__name__", None):
-                            KNOWN_FUNCTION_LOCATIONS[f] = thing.__name__
-                except (TypeError, ValueError):
-                    pass
         else:
             raise InvalidArgument(f"Can't test non-module non-callable {thing!r}")
 
+        for f in copy(funcs):
```

Comment:
Instead of importing `copy()`, we can call `list(funcs)` here - it's equivalent as a way to make a shallow copy, with a bit less code.

Reply:
ok